gce/firewall.tf

```diff
@@ -12,6 +12,25 @@ resource "google_compute_firewall" "ssh" {
   }
 }
 
+resource "google_compute_firewall" "bosh" {
+  name = "${var.env}-cf-bosh"
+  description = "Allow bosh deployed vms to route via bastion"
+  network = "${google_compute_network.bastion.name}"
+
+  source_tags = [ "cf" ,"bastion", "bosh"]
+  target_tags = [ "bastion", "cf", "bosh" ]
+
```

Comment:
would be easier to see that the source and target tags are the same if the order of tags in the lists was the same...